nightly/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   master:
```

Comment:
`s/master/coordinator/`

Reply:
Should we rollback because it is already on master